build-linux/deb/postinst

```diff
@@ -53,9 +57,11 @@ if [ "$1" = "configure" ]; then
     # outqueue/* world permission is set to 000 so that service keys of other
     # users cannot be seen, but set to -wx for enqueue-related directories
     # to allow any user to write events.
```

Comment:
@dobs wondering if I can get some help with this comment above. Technically world permission is set to 7 now but anytime it wasn't set to 7 (for example just 6), I'd get errors from systemd saying the necessary files weren't writeable.